### ISSUE TITLE
Logo example - CGB rom shows malformed image on DMG

### ISSUE THREAD

**Issue by brettinman** [gbdk-2020/gbdk-2020#618]
Hi, I've been playing around with making a logo cartridge based on the logo example.

When I use the included example logo and the CGB output rom, it displays properly on both a CGB emulator/hardware and on DMG emulator/hardware (just in grayscale). Based on this I think it is processing the PNG correctly and displaying it correctly to be backwards-compatible on non-Color systems.

However, when I replace the logo with my own PNG and build the CGB rom, it shows the correct image on CGB emulators/hardware but the image is slightly mangled on DMG emulators and hardware. I've tried several images to figure out if it was based on the number of colors, types of colors (eventually I tried just black and white), complexity of image, etc - but all are mangled.

I'm not sure what makes the example logo special - am I missing some property of the image itself, or does some of the code need to be modified to accommodate a different image (palettes, tiles, etc)?

Test PNG:
![basic](https://github.com/gbdk-2020/gbdk-2020/assets/5668414/ad35155d-3182-4570-aac1-e80149c00972)

Screenshot from Gameboy Color mode in BGB:
![CGB](https://github.com/gbdk-2020/gbdk-2020/assets/5668414/c2cef6ca-933d-4d09-9860-b5c33588f7e7)

Screenshot from Gameboy mode in BGB:
![DMG](https://github.com/gbdk-2020/gbdk-2020/assets/5668414/069298ea-7acf-42d4-ae4e-f753bae2c250)

**Comment by untoxa** [comment]
you must pass -noflip to disable flipping for the background tiles, because that is not supported by the DMG. it is not the GBDK-2020 issue.

**Comment by brettinman** [comment]
I did not mean to imply it was a bug with gbdk, was just looking for guidance on how to use the example project and could not find any previous info on the topic.

Your suggestion was correct, using noflip fixed the problem. Thank you for the help! 

I found more info on noflip here if it helps anyone in the future: https://laroldsjubilantjunkyard.com/tutorial/png2asset-graphics-creation-for-gbdk-2020/

If you don't mind, I'll open a PR to update the example readme with this information to help future users.
